src/EnergyPlus/CostEstimateManager.cc

```diff
@@ -683,6 +694,85 @@ namespace CostEstimateManager {
                         }
                     }
 
+                } else if (SELECT_CASE_var == "COIL:COOLING:DX") {
+
+                    auto &parentObjName = state.dataCostEstimateManager->CostLineItem(Item).ParentObjName;
+                    bool coilFound = false;
+                    thisCoil = 0;
+                    //  check for wildcard * in object name..
+                    if (parentObjName == "*") { // wildcard, apply to all such components
+                        WildcardObjNames = true;
```

Comment:
... following WildcardObjNames reset from above, this bool will be false getting here, right? Meaning if a previous coil type had a wild card, this variable will get reset before looking for the new coil model.

Reply:
Yes, the reset above should ensure it's always false before it gets here.